src/tracker/tracker.py

```diff
@@ -50,6 +51,7 @@ def __init__(self, cfg: ReIDObjetcTrackerConfig, camera, debug: bool = False):
         self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
         self.detector: Detector = get_detector(cfg.detector_config)
+        self.ir_detector: Detector = IRDetector(cfg.detector_config)
```

Comment:
almost good - there is an issue here. the IR Detector can have different config attributes than the RGB detector. 
For instance `threshold`. Hence you need to create a new config class `IRDetectorConfig` to be able to configure independently